tests/core/full_node/stores/test_coin_store.py

```diff
@@ -552,12 +552,14 @@ def random_coin_records() -> RandomCoinRecords:
 @pytest.mark.parametrize("include_spent", [True, False])
 @pytest.mark.parametrize("include_unspent", [True, False])
 @pytest.mark.parametrize("include_hinted", [True, False])
+@pytest.mark.parametrize("min_amount", [uint64(0), uint64(30000)])
```

Comment:
I would use some uncomfortable values here. Like, `0xffff`, `0x7fff`, `0x8000`, or `0x8000000000000000`

Reply:
One of my main concerns with this is that the number of combinations of things to test can blow up exponentially - maybe it's worth it in this case

Reply:
Didn't affect it that much, just added these values